definitions/tools/concordance.cwl

```diff
@@ -1,48 +1,44 @@
+#!/usr/bin/env cwl-runner
+
 cwlVersion: v1.0
 class: CommandLineTool
-label: "Concordance tool"
-baseCommand: ["python", "/opt/concordance/newConcordance.py"]
+label: "Concordance checking between Tumor and Normal BAM"
+baseCommand: ["/usr/bin/somalier"]
 requirements:
-    - class: DockerRequirement
-      dockerPull: "mgibio/concordance"
     - class: ResourceRequirement
-      ramMin: 4000
+      coresMin: 1
+      ramMin: 8000
+      tmpdirMin: 10000
+    - class: DockerRequirement
+      dockerPull: "brentp/somalier:v0.1.5"
+arguments: ["-o", "concordance"]
 inputs:
-    bam_1:
+    vcf:
         type: File
         inputBinding:
+            prefix: "-s"
+            position: 1
+    reference:
+        type: string
+        inputBinding:
+            prefix: "-f"
             position: 2
-        secondaryFiles: [.bai]
-    bam_2:
+    bam_1:
         type: File
         inputBinding:
             position: 3
         secondaryFiles: [.bai]
-    reference:
+    bam_2:
         type: File
         inputBinding:
             position: 4
-        secondaryFiles: [.fai, "^.dict"]
-    snp:
-        type: File
-        inputBinding:
-            position: 5
-    output_file_name:
-        type: string?
-        inputBinding:
-            position: 6
-        default: "concordance_report.txt"
-    output_geno_file_name:
-        type: string?
-        inputBinding:
-            position: 7
-            prefix: "--output_geno" 
+        secondaryFiles: [.bai]
 outputs:
-    output_file:
+    somalier_pairs:
         type: File
         outputBinding:
-           glob: "$(inputs.output_file_name)"
-    output_geno_file:
-        type: File?
+            glob: "concordance.somalier.pairs.tsv"
```

Comment:
Just a note here and probably out of the scope of this PR. It would be better to add some specific identifier prefix to this hardcoded output name.

Reply:
Yeah, none of this somatic pipeline includes sample names in file names, IIRC.  I'll say that's an issue for another PR.